src/components/elements/Loading/Loading.tsx

```diff
@@ -1,40 +1,33 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { Segment } from 'semantic-ui-react';
+import type { SegmentProps } from 'semantic-ui-react';
 import LoadingOverlay from '../LoadingOverlay';
 
-/**
- * Loading is a block component which uses LoadingOverlay component
- * to display loader in center of parent component.
- */
-// @ts-expect-error TS(7031) FIXME: Binding element 'message' implicitly has an 'any' ... Remove this comment to see the full error message
-function Loading({ message, style, className }) {
-    return (
-        <Segment basic style={{ height: '100%', zIndex: 5, ...style }} className={className}>
-            <LoadingOverlay message={message} />
-        </Segment>
-    );
-}
-
-Loading.propTypes = {
+export interface LoadingProps extends SegmentProps {
```

Comment:
It seems like we don't need to use `SegmentProps` interface as all used props are being typed by hand 💪 

```suggestion
import LoadingOverlay from '../LoadingOverlay';

export interface LoadingProps {
```

Reply:
https://github.com/cloudify-cosmo/cloudify-ui-components/pull/170/commits/d5a17b4573be3d4b77685ec2940920dafef52660 removed SegmentProps :)